galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -535,3 +535,16 @@ messages:
     description: "user namespace should be injectable if Istio is installed with enableNamespacesByDefault enabled and neither injection label is set."
     template: "is enabled for Istio injection, as Istio is installed with enableNamespacesByDefault as true."
     url: "https://istio.io/latest/docs/reference/config/analysis/ist0148/"
+
+  - name: "JwtClaimBasedRoutingWithoutRequestAuthN"
+    code: IST0149
+    level: Error
```

Comment:
Is this condition always an error?  Is it possible to use this header for some other purpose, which would trigger this analyzer?

Reply:
it shouldn't, the `@` prefix already makes it an invalid HTTP header, if a customer puts it in virtual service today, it will not work at all, and Envoy will simply reject the header with HTTP code 400.